### PR TITLE
Added test preference repository

### PR DESCRIPTION
**Pull request by ArtemAstakhov**

### CODE REVIEW

File: bot/Cargo.toml [hunk 1]
```diff
@@ -15,4 +15,5 @@ rust-i18n = { version = "2", default-features = false, features = [] }
 sqlx = { version = "0.7", features = [ "runtime-tokio", "postgres" ] }
 async-trait = { version = "0.1"}
 electricity = { path = "../electricity" }
-tracing = { workspace = true }
\ No newline at end of file
+tracing = { workspace = true }
+chrono = { version = "*"}
```
Comment (zolkko) [inline bot/Cargo.toml:19]: Set 0.4 version.

File: bot/src/message_handler.rs [hunk 2]
```diff
@@ -128,25 +127,28 @@ async fn get_sqlx_database_client() -> Result<sqlx::PgPool> {
...
-fn get_update_language_code(update: &Update) -> String {
-    let mut language_code = "en".to_string();
+fn get_update_language_code(update: &Update) -> Language {
+    let mut language_code = Language::En;
 
     if let UpdateKind::Message(message) = &update.kind {
         if let Some(user) = &message.from() {
             if let Some(language_code_option) = &user.language_code {
-                language_code = language_code_option.to_string();
+                language_code = Language::from_str(language_code_option).unwrap_or(Language::En);
```
Comment (zolkko) [inline bot/src/message_handler.rs:136]: Let's keep it as it is. But later we may add a warning to the log.

File: bot/src/message_handler.rs [hunk 3]
```diff
@@ -155,15 +157,10 @@ async fn get_chat_preference(
...
-        } else {
-            "en".to_string()
-        };
 
         let new_chat_preference = NewChatPreference {
             chat_id,
-            language: std::str::FromStr::from_str(&language_code)?,
+            language: language_code,
         };
         chat_preference_repository
```
Comment (zolkko) [inline bot/src/message_handler.rs:165]: Is it missing await?

Comment (ArtemAstakhov) [reply bot/src/message_handler.rs:165]: hm, it has it

File: bot/src/message_handler.rs (hunk 2 above)
Comment (ArtemAstakhov) [reply bot/src/message_handler.rs:136]: I didn't get that point

File: bot/Cargo.toml (hunk 1 above)
Comment (ArtemAstakhov) [reply bot/Cargo.toml]: fixed

File: bot/src/message_handler.rs (hunk 3 above)
Comment (zolkko) [reply bot/src/message_handler.rs:165]: ❤️

**Pull request merged**